R/crimescript.R

```diff
@@ -0,0 +1,85 @@
+# Load necessary libraries
+library(tidyverse)
+library(readxl)
+library(devtools)
+
+# File paths
+ukcrime_file <- 
+  "C:/Users/ZaraMorgan/Downloads/ukcrimetables.xlsx"
+ukcrime_rda <- 
+  "C:/Users/ZaraMorgan/Downloads/ukcrimetables.rds"
+lookup_file <- 
+  "C:/Users/ZaraMorgan/OneDrive - British Red Cross Society/Local_Authority_District_to_CSPs_to_Police_Force_Areas_(December__2023)_Lookup_.xlsx"
+
+# Load the UK crime table and save as RDA
+ukcrimetables <- read_excel(ukcrime_file, sheet = 8, 
+                            skip = 7)
+save(ukcrimetables, file = ukcrime_rda)
```

Comment:
It is not necessary to read, save and load the dataset. You only need to read the dataset to be able to manipulate it in R.